src/Dfe.PlanTech.Application/Submission/Commands/SubmitAnswerCommand.cs

```diff
@@ -72,7 +72,30 @@ public async Task<bool> NextQuestionIsAnswered(int submissionId, string nextQues
             return false;
         }
 
-        public async Task<Domain.Questionnaire.Models.Question?> GetNextUnansweredQuestion(int submissionId)
+        public async Task<(Domain.Questionnaire.Models.Question? Question, Domain.Submissions.Models.Submission? Submission)> GetNextQuestion(
```

Comment:
I'm a bit confused around this method; it's called `GetNextQuestion` but it returns a question and a submission. There's also a method just below it of `_GetNextUnansweredQuestion` (which this also calls) which adds to the confusion.

Is there anything that we can do to make this a bit more readable?

Reply:
Oops, yeah I had a bit of trouble coming up with the name of this method so instead of thinking about it too much I just sort of named it with the intention of coming back later and giving it a better name.. Which I forgot about lol

Still not sure what the nicest name for it would be, maybe just something like `GetQuestionWithSubmission()`?

Reply:
Renamed in: [8b49c58](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/216/commits/8b49c5809780e8330d4b0a91c08baa07a8c29d06)